haiku-apps/fairtrade/fairtrade-1.0.0.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A keyboard only app for fair trade sellers"
```

Comment:
Hmm, I found a better summary compared to the one I had suggested.
It's taken from the [project homepage](https://github.com/czeidler/fairtrade).
```
SUMMARY="A simple to use point of sale system and product management"
```

BTW, you forgot to split DESCRIPTION into lines <= 80 chars.

Regarding the name of the recipe, maybe it should be changed to fairtrade-0.0.0~git_2014.06.22.recipe or something similar.
@korli, what name should we use?